utils/appliance/implementations/ui.py

```diff
@@ -137,9 +137,11 @@ def create_view(self, *args, **kwargs):
 
     def am_i_here(self):
         try:
-            return self.view.is_displayed
-        except (AttributeError, NoSuchElementException):
+            self.view.assert_is_displayed()
```

Comment:
Yes, views should be changed, because `is_displayed` tell you nothing about why view is not displayed. But if we have both `is_displayed` and `assert_is_displayed()` it will double the code.